editor-components/feature/feature12/feature12.module.scss

```diff
@@ -1,52 +1,229 @@
 .container {
     width: 100%;
+    height:100%;
     padding: 50px 0;
+    background-color: #0c0f16;
+    padding-bottom:30px;
+    
+    .upper-title {
+        padding-bottom: 20px;
+        color: #46caa5;
+        letter-spacing: 10px;
+        position: relative; 
+
+        z-index:5;
+        transform: translateY(82%);
+
+    }
+    .shadow-header-title {
+        position: absolute;
+        color: #0c0f16;
+        text-shadow: -1px -1px 0 #1b1b1b,  
+                     1px -1px 0 #1b1b1b,
+                     -1px 1px 0 #1b1b1b,
+                     1px 1px 0 #1b1b1b;
+        z-index: 0;
+        font-size: 95px; 
+        width: 100%;
+        text-align: center;
+        transform: translateY(-30px);
+    }
+    
+    .header-title {
+        position: absolute;
+        color: #FFFFFF;
+        padding-bottom: 20px;
+        letter-spacing: 3px;
+        text-shadow: 
+        1px 1px 0 #2a2929,  
+        2px 2px 0 #2a2929, 
+        3px 3px 0 #2a2929;
+        font-weight: bold;
+        font-size: 100px; 
+        z-index: 2; 
+        width: 100%;
+        text-align: center;
+        transform: translateY(20%);
+
+    }
     .max-content {
+        margin-top: 100px;
         max-width: var(--composer-content-width);
         margin: auto;
         display: flex;
-        justify-content: space-around;
+        padding-left: 10px;
+        width: 75%;
+        padding-right: 10px;
+        height: 100%;
+        justify-content: center;
+
         .content {
+            margin-top: 110px;
             display: flex;
-            justify-content: center;
+            justify-content:center;
             flex-wrap: wrap;
-            gap: 20px;
+            width:100%;
+            height: 100%;
+            gap:0px;
+
+            .card-item-first,
+            .card-item-count {
+                min-height:50vh;
+            
+                margin-top: 60px;
+                
+            }
+            
+
+            .card-item-first {
+            
+                width: 100%;
+                align-items: center;
+                background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.75)), url('https://avo.themescamp.com/wp-content/uploads/2020/09/1-2.jpg');
```

Comment:
if this value does not come from the prop the user cannot change the background image